lib/nylas/event.rb

```diff
@@ -43,6 +43,7 @@ class Event
     attribute :reminder_minutes, :string
     attribute :reminder_method, :string
     attribute :job_status_id, :string, read_only: true
+    attribute :visibility, :string, read_only: true
```

Comment:
Visibility is actually writeable for Google events (which is the only provider that uses this field).

```suggestion
    attribute :visibility, :string
```